gui/vm/forms.py

```diff
@@ -97,13 +123,34 @@ class DeviceForm(ModelForm):
         required=False,
         dirsonly=False,
     )
+    DISK_raw_boot = forms.BooleanField(
+        label=_('Disk boot'),
+        widget=forms.widgets.HiddenInput(),
+        required=False,
+        initial=False,
+    )
+    ROOT_password = forms.CharField(
+        label=_('Password'),
+        max_length=50,
+        widget=forms.widgets.HiddenInput(),
+        required=False,
+        help_text=_("Set the user rancher password."),
+    )
     DISK_sectorsize = forms.IntegerField(
         label=_('Disk sectorsize'),
         required=False,
         initial=0,
         help_text=_("Logical and physical sector size in bytes of the emulated disk."
```

Comment:
"Sector size of the emulated disk in bytes. Both logical and physical sector size are set to this value."